Keywords.txt

```diff
@@ -1,3 +1,4 @@
+Sweltered | Increases damage NPCs take by thirty-five percent, while also dealing twenty DoT. | 255 | 100 | 35
```

Comment:
Vanilla never spells out numbers like this, lets make it more consistent. Lets also not use the abbreviation DoT.